components/net/http_loader.rs

```diff
@@ -2015,6 +2024,7 @@ async fn http_network_fetch(
                 meta_headers.map(Serde::into_inner),
                 meta_status,
                 pipeline_id,
+                browsing_context_id,
```

Comment:
Let's only send the information to the devtools if there is a non-None browsing context id.